### ISSUE TITLE
Autocomplete Keyboard Accessibility Does Not Work in Non-Chrome Browsers

### ISSUE THREAD

**Issue by mlaursen** [mlaursen/react-md#137]
Using the up and down keys does not actually select new items in Firefox or IE.
